### PR TITLE
Change user (eye) userHeight from 1.8 to 1.6

### LINKED ISSUE #1717: Consider lowering default userHeight a tad.
**Description:**

From googling around, the average eye height is 1.44 meters (inclusive female + male). We currently set to 1.8.

- A-Frame Version:
- Platform/Device:

### PR DESCRIPTION
**Pull request by mkungla**
closes #1717

### CODE REVIEW

**Comment by ngokevin** [comment]
Thanks! Looks good. Waiting for tests to pass.

**Comment by mkungla** [comment]
yeah travis had false alarm there due (network timeout) and is now re running the tests.

**Comment by ngokevin** [comment]
Yup, I restarted it.

File: src/extras/primitives/primitives/a-camera.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ var registerPrimitive = require('../primitives').registerPrimitive;
 
 registerPrimitive('a-camera', {
   defaultComponents: {
-    camera: {},
+    camera: {
```
Comment (dmarcos) [inline src/extras/primitives/primitives/a-camera.js]: one liner here?

**Comment by mkungla** [comment]
aight will commit

File: src/extras/primitives/primitives/a-camera.js (hunk 1 above)
Comment (mkungla) [inline src/extras/primitives/primitives/a-camera.js]: @dmarcos across other primitives single values are not

**Pull request merged**